src/schemas/transaction.yaml

```diff
@@ -356,8 +359,20 @@ UserOperation:
       title: max fee per gas
       description: The maximum total fee per gas the sender is willing to pay (includes the network / base fee and miner / priority fee) in wei
       $ref: '#/components/schemas/uint'
-    paymasterAndData:
-      title: paymaster concatenated with paymaster data
+    paymaster:
```

Comment:
1. need to make all paymaster fields optional
2. need to split initCode too (see the erc)
stretch:
3. if possible with schema: all paymaster fields should be included, or none. We do have a code to test it, but it would be nice if the schema could enforce that the optional paymaster can't be included unless also paymasterData and the rest are also there.

Reply:
1. Done
2. Done
3. I am not familiar with this schema language but it seems pretty linear, would be surprised if something like that is possible. For example, `GenericTransaction` lists all of 'gasPrice', 'maxPriorityFeePerGas' and 'maxFeePerGas' seemingly without defining such relations.

Reply:
It can be done with "allOf/anyOf/if/then" combinations